### PR TITLE
Feature/gcloud storage functions

### PR DESCRIPTION
**Pull request by ju4nlu**
- I've upgraded the version number to `0.5.0` because this PR includes a new class and functionality. I think that's enough reason to bump the version number in this way.
- There are no tests relative to those methods that demand a working internet connection.

### CODE REVIEW

**Comment by nradz** [comment]
If there is a new feature, you have to upgrade the minor version. So, it is ok :+1: 
https://semver.org/

File: SwissKnife/gcloud/GCloudStorage.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        :type with_prefix: bool, optional
+        :return: Complete path of a file stored in gcloud
+        :rtype: str
+        """
+        bucket_name = f'{BUCKET_NAME}/' if with_bucket else ''
+        file_prefix = f'{BUCKET_PATH_PREFIX}/' if with_prefix and BUCKET_PATH_PREFIX else ''
+        
+        return f'{bucket_name}{file_prefix}{file_path}/{file_name}'
+
+    def list_blobs(self, storage_path: str):
```
Comment (nradz) [inline SwissKnife/gcloud/GCloudStorage.py]: The return type is missing.

File: SwissKnife/gcloud/GCloudStorage.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            blob.upload_from_filename(data)
+        else:
+            raise NotImplementedError
+        
+        file_path = self._get_storage_file_path(destionation_storage_path, destionation_file_name, with_bucket=True)
+        
+        self.logger.info(f"Uploaded file to gcloud path {file_path}")
+        return file_path
+        
+    def __generate_blob(self,
```
Comment (nradz) [inline SwissKnife/gcloud/GCloudStorage.py:114]: The return type is missing. If it is complicated to defined the return type, you can set a descriptive string:

`def _generate_blob(...) -> "blob":`

**Review by nradz — APPROVED** [review]

**Review by eoterogarcia — APPROVED** [review]

File: SwissKnife/gcloud/GCloudStorage.py (hunk 2 above)
Comment (ju4nlu) [reply SwissKnife/gcloud/GCloudStorage.py:114]: Nice, I'll do that

File: SwissKnife/gcloud/GCloudStorage.py (hunk 1 above)
Comment (ju4nlu) [reply SwissKnife/gcloud/GCloudStorage.py]: Yeah, it returns an iterator and I was wondering how to annotate it, I'll use the trick mentioned below.

**Pull request merged**